src/hooks/useData.ts

```diff
@@ -0,0 +1,28 @@
+'use client';
+import { useBearStore } from '@/store/store';
+import { server } from '@/utils/server';
+import { useEffect } from 'react';
+
+export default function useData() {
```

Comment:
Avoid using data words, everything's data. Let's add a more descriptive name for this hook, please.